device_plugins/deploy_qat.md

```diff
@@ -60,10 +60,10 @@ Users can use the steps below to customize the QAT resource configuration:
     ```
     Options:  
     `dc`: Configure all the QAT VF devices managed by the device plugin CR for compression/decompression.  
-    `sym` or `asym`: Configure all the QAT VF devices managed by the device plugin CR for cryptography 
+    `sym`;`asym`: Configure all the QAT VF devices managed by the device plugin CR for cryptography 
```

Comment:
`sym;asym`

Reply:
@vbedida79  please also confirm this.

Reply:
same in https://github.com/intel/intel-device-plugins-for-kubernetes/blob/v0.26.2/demo/qat-init.sh @mregmi is this right

Reply:
Yep Looks good.

Reply:
LGTM